public/components/FeatureAnywhereContextMenu/Container/Container.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import './styles.scss';
+import AssociatedDetectors from '../AssociatedDetectors/containers/AssociatedDetectors';
+
+const Container = ({ startingFlyout, ...props }) => {
```

Comment:
nit - can we have this named something else to be more specific here? `Container` is a bit vague.

Reply:
Does `AnywhereParentFlyout` sounds good?

Reply:
fine with me :)